chia/wallet/wallet_node.py

```diff
@@ -729,15 +740,20 @@ async def state_update_received(self, request: wallet_protocol.CoinStateUpdate,
             )
 
     def get_full_node_peer(self) -> Optional[WSChiaConnection]:
-        assert self.server is not None
+        if self.server is None:
+            return None
+
         nodes = self.server.get_full_node_connections()
         if len(nodes) > 0:
             return random.choice(nodes)
         else:
             return None
 
     async def disconnect_and_stop_wpeers(self):
-        # Close connection of non trusted peers
+        if self.server is None:
+            return False
```

Comment:
The existing return hint for this function would have been `None`.  Do we want to change it to `bool` or should this just be `return`?